docs/user/integration/assets/cloudwatch.drawio.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please make sure your diagram adheres to our guidelines: https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md